README.md

```diff
@@ -776,8 +776,7 @@ A curated list of awesome Java frameworks, libraries and software.
 * [Fixture Factory](https://github.com/six2six/fixture-factory) - Generates fake objects from a template.
 * [JGiven](http://jgiven.org) - Developer-friendly BDD testing framework compatible with JUnit and TestNG.
 * [JMockit](http://jmockit.org/) - Mocks static, final methods and more.
-* [JSONAssert](http://jsonassert.skyscreamer.org/) - Simplifies testing of JSON 
-s.
+* [JSONAssert](http://jsonassert.skyscreamer.org/) - Simplifies testing of JSONs.
```

Comment:
Hi @shekhargulati, sorry if I wasn't clear: could you completely revert your change here? IMO "JSON strings" sounds more idiomatic than "JSONs", as "JSONs" isn't a term I've heard or seen anywhere before.